src/main/java/com/github/hcsp/datatype/Cast.java

```diff
@@ -4,48 +4,69 @@
     // Cast an int to byte
     // 将int类型转换成byte
     public static byte int2byte(int i) {
+        int a=10;
+        byte i=(byte) a;
```

Comment:
```suggestion
        byte b=(byte) i;
```

`i`变量重复定义了。下面同理。

另外你声明个`int a = 10`是啥意思？